### PR TITLE
Update about.html.haml

### PR DESCRIPTION
**Pull request by Riyuzakii**
I went through your website and you have given the link for everything so, I added the link of google grants page here.

### CODE REVIEW

**Comment by Riyuzakii** [comment]
@equivalentideas So can this be merged now.

File: app/views/static/about.html.haml [hunk 1]
```diff
@@ -35,3 +35,8 @@
   = link_to "useiconic.com", "https://useiconic.com/"
   from
   #{link_to "the Noun Project", "http://thenounproject.com/"}.
+  
+  %p
+    PlanningAlerts receives free access to use Google Maps services through the&nbsp;
+    %a(href="https://www.google.com/grants/")> Google Grants Program
```
Comment (equivalentideas) [inline app/views/static/about.html.haml]: This link should use the `link_to` Rails method, like the one on [line 35 above](https://github.com/openaustralia/planningalerts/pull/1107/files#diff-df7147916bb65cbad52262ae3a631dcfR35). See http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to.

File: app/views/static/about.html.haml [hunk 2]
```diff
@@ -35,3 +35,8 @@
   = link_to "useiconic.com", "https://useiconic.com/"
   from
   #{link_to "the Noun Project", "http://thenounproject.com/"}.
+  
+  %p
+    PlanningAlerts receives free access to use Google Maps services through the&nbsp;
```
Comment (equivalentideas) [inline app/views/static/about.html.haml]: What's the `&nbsp;` on the end of this line for?

**Comment by equivalentideas** [comment]
@Riyuzakii 😄 how did you find out about PlanningAlerts and what made you want to contribute?

**Comment by Riyuzakii** [comment]
Well, I was just going through the easy fix bugs on github and i found planning alerts. Then i visited the website and i think it's a pretty unique idea.

**Comment by equivalentideas** [comment]
Closing this one now that https://github.com/openaustralia/planningalerts/pull/1109 is open.

**Pull request closed without merging**